### DISCUSSION TITLE
Is there a shared folder between my frontend and /api?

### DISCUSSION THREAD

**Discussion by cosbgn** [vercel/vercel#4248] [General] [answered]
I have a few json schema files which I would like to use in both my frontend and my `/api` backend functions. This is to validate data directly client and server side.

Is there a way to share a folder between my fronend and my backend `/api` folder? Right now I have a `prebuild` script which copies them but it's not handy during development.

**Comment by coetry** [comment, marked as answer]
Hi @Cosbgn you can use `fs.readFileSync` in order to let our static analysis tool know that your function depends on a file.

We have some documentation on that [here](https://vercel.com/docs/runtimes#advanced-usage/technical-details/including-additional-files).
